aggregate_temporal.json

```diff
@@ -150,80 +157,61 @@
         }
     ],
     "returns": {
-        "description": "A data cube with potentially lower resolution and potentially lower cardinality, but the same number of dimensions as the original data cube.",
+        "description": "A data cube with the same dimensions. The dimension properties (name, type, labels, reference system and resolution) remain unchanged, except for the resolution and dimension labels of the given temporal dimension.",
         "schema": {
             "type": "object",
             "subtype": "raster-cube"
         }
     },
     "examples": [
         {
-            "process_graph": {
-                "loadco1": {
-                    "process_id": "load_collection",
-                    "arguments": {
-                        "id": "Sentinel-2",
-                        "spatial_extent": {
-                            "from_parameter": "spatial_extent"
-                        },
-                        "temporal_extent": [
-                            "2015-01-01",
-                            "2019-01-01"
-                        ]
-                    }
+            "arguments": {
+                "data": {
+                    "from_parameter": "data"
                 },
-                "aggreg1": {
-                    "process_id": "aggregate_temporal",
-                    "arguments": {
-                        "data": {
-                            "from_node": "loadco1"
-                        },
-                        "intervals": [
-                            [
-                                "2015-01-01",
-                                "2016-01-01"
-                            ],
-                            [
-                                "2016-01-01",
-                                "2017-01-01"
-                            ],
-                            [
-                                "2017-01-01",
-                                "2018-01-01"
-                            ],
-                            [
-                                "2018-01-01",
-                                "2019-01-01"
-                            ]
-                        ],
-                        "labels": [
-                            "2015",
-                            "2016",
-                            "2017",
-                            "2018"
-                        ],
-                        "reducer": {
-                            "process_graph": {
-                                "median1": {
-                                    "process_id": "mean",
-                                    "arguments": {
-                                        "data": {
-                                            "from_parameter": "data"
-                                        }
-                                    },
-                                    "result": true
+                "intervals": [
+                    [
+                        "2015-01-01",
```

Comment:
Minor comment (feel free to ignore):

Maybe a bit late, but perhaps shift the date range by one year towards 2020 for a better potential data coverage with Sentinel-2?

Reply:
The example is not specific to any data set (any longer), but if you think that would give a user a benefit, I'm happy to add a year.

Reply:
Added

Reply:
Whatever makes sense - my motivation was that user may try it with Sentinel-2. No strong opinion here!